pkg/services/ngalert/state/manager.go

```diff
@@ -33,7 +33,7 @@ func NewManager(logger log.Logger, metrics *metrics.Metrics, ruleStore store.Rul
 	manager := &Manager{
 		cache:         newCache(logger, metrics),
 		quit:          make(chan struct{}),
-		ResendDelay:   1 * time.Minute, // TODO: make this configurable
+		ResendDelay:   10 * time.Second, // TODO: make this configurable
```

Comment:
This change doesn't mean that notifications will be sent once per elapsed 10 seconds instead of once per minute right? This is just the interval at which we resend the alert to Alertmanager which will then itself choose whether to re-send the notification or not?

Reply:
@gerobinson Correct, it is like a keep-alive msg from the state manager to alert manager (since in AM, if endsAt is passed, the alert instance is resolved).